docs/high-availability.md

```diff
@@ -0,0 +1,193 @@
+# High Availability Kubernetes Clusters
+
+## Introduction
+This document describes how to build a high-available Kubernetes cluster.  This is a fairly advanced topic.
+Users who merely want to experiment with Kubernetes are encouraged to use configurations that are simpler to set up such as
+the simple [Docker based single node cluster instructions](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/docker.md),
+or try [Google Container Engine](https://cloud.google.com/container-engine/) for hosted Kubernetes.
+
```

Comment:
Not only are they simpler, but they are continuously tested.  I'm sure we'll get to the point where the HA is part of e2e, but until then, best to make it clear which path is best supported.